### ISSUE TITLE
Challonge::Tournament#publish! has no server-side endpoint anymore

### ISSUE THREAD

**Issue by arielkirkwood** [challonge-community/challonge-ruby-gem#3]
The `#publish!` method seems to have been deprecated from the API, since [the API page](http://api.challonge.com/v1) no longer lists a `publish` endpoint to POST to.

Looks like it should be removed from the tournaments module.

The README example should be updated to correctly communicate what an API consumer can expect after saving a tournament. Are `t.live_image_url` and `t.full_challonge_url` available after saving instead of after publishing now?

There may be more to do regarding this issue or more generally, bringing the client up to date with the current version of the Challonge API - I'm a novice Rubyist and am willing to help, if someone could perhaps direct me.

**Comment by yjukaku** [comment]
@arielkirkwood Yep, I just confirmed this with the Challonge team. `#publish!` needs to be removed from the documentation.

**Comment by arielkirkwood** [comment]
@yjukaku Thanks for looking into it. Let me know if I can help out with anything.

**Comment by arielkirkwood** [comment]
Whoops, I just realized you aren't a maintainer of this repo (at least it doesn't look like it). Have you talked to any of the other contributors recently?

**Comment by akdarrah** [comment]
@arielkirkwood @yjukaku Thanks for investigating this. I should be able to get this updated tonight.

**Comment by corneldm** [comment]
And yes, we're in the loop. @akdarrah and I will be clearing away some cobwebs and hopefully adding new API endpoints while we're at it (MatchAttachment, check-in related actions, ...)

**Comment by akdarrah** [comment]
@arielkirkwood Thanks for pointing this out. I removed the publish! method and added finalize!, which should be all the methods required to run a basic tournament (and the README has been updated too).

`t.live_image_url` and `t.full_challonge_url` are both available after starting the tournament.

We still need to update the gem version and add support for check-in endpoints, so this issue should stay open until those things have been taken care of.

**Comment by khisakuni** [comment]
Looks like the gem version hasn't been updated yet. Any progress on that?
